libraries/hatch-client-sdk/src/hatch-client-sdk.ts

```diff
@@ -0,0 +1,23 @@
+import {createClientSDKByDependency, createClientSDKByInputSpec} from './util';
+
+const usage = 'Usage: hatch-client-sdk [ --input input-spec | --dependency dependency-name ]';
+const argv = process.argv.slice(2);
+const typeArg = argv[0];
+if (argv.length < 2) {
+  console.error('Invalid arguments');
+  console.log(usage);
```

Comment:
Should we be throwing or exiting with an error code here and elsewhere?

Reply:
updated